src/mono/wasm/runtime/jiterpreter-support.ts

```diff
@@ -901,6 +909,8 @@ export type JiterpreterOptions = {
     dumpTraces: boolean;
     // Use runtime imports for pointer constants
     useConstants: boolean;
+    // Punch through gsharedvt wrappers when compiling jitcall wrappers if possible
+    punchThrough: boolean;
```

Comment:
This could use a different name, like 'direct calls' or something.

Reply:
OK, if it's confusing I'll rename it.